synapse-grid-core/src/main/scala/ru/primetalk/synapse/core/runtime/SignalProcessingApi0.scala

```diff
@@ -119,6 +119,11 @@ trait SignalProcessingApi0 extends SignalsApi with TrellisApi with RuntimeCompon
                 val (ns, signals) = f.asInstanceOf[(Any, Signal[_]) => (Any, List[Signal[_]])](s, signal)
                 trellisBuilder.currentState = trellisBuilder.currentState.asInstanceOf[Map[Contact[Any], Any]].updated(sh, ns).asInstanceOf[Context]
                 trellisBuilder.addSignals(trace, proc, signals)
+              case BlackBoxRuntimeComponent(_, _, _, f) =>
```

Comment:
I thought that if it's `RuntimeComponent` there should be a branch that supports that interface. If not, we could replace this with
```
case rc: RuntimeComponent =>
```

Reply:
Done.